app/templates/components/structural/project-list-map.hbs

```diff
@@ -17,13 +17,15 @@
       }}
 
       {{!-- Anything else for dynamic tiles, like hovers --}}
-      {{dynamic-tile-source.layer 
+      {{dynamic-tile-source.layer
         layer=projectCentroidsCircleHoverLayer
       }}
 
-      {{dynamic-tile-source.layer 
+      {{dynamic-tile-source.layer
         layer=projectPolygonsHoverLayer
       }}
+
+      {{map-info-box}}
```

Comment:
@andycochran can you find a way to move this back where it was? I want project-list-map to be a "provider" component and not really render any dom.